x-pack/plugins/triggers_actions_ui/public/common/get_alerts_table_state.tsx

```diff
@@ -5,21 +5,28 @@
  * 2.0.
  */
 
+import React, { lazy, Ref, Suspense } from 'react';
 import { EuiLoadingSpinner } from '@elastic/eui';
-import React, { lazy, Suspense } from 'react';
-import { LazyLoadProps } from '../types';
+import { typedForwardRef } from '../../common/utils';
+import {
+  AdditionalContext,
+  AlertsTableImperativeApi,
+  AlertsTableProps,
+  LazyLoadProps,
+} from '../types';
+import { type AlertsTable } from '../application/sections/alerts_table/alerts_table';
 
-import type { AlertsTableStateProps } from '../application/sections/alerts_table/alerts_table_state';
+const AlertsTableStateLazy = lazy(
```

Comment:
Yes, but since when moving the table to a package I removed all these getters I preferred to avoid potential problems with the renaming since `getAlertsTableState...` takes the place of the already existing `getAlertsTable...`. I can do it though, it shouldn't be too hard to check!

Reply:
No worries. Let's revise later. When we see the whole picture in the feature PR we can decide better.